app/web/src/store/feature_flags.store.ts

```diff
@@ -95,6 +95,7 @@ export function useFeatureFlagsStore() {
         // You can override feature flags while working on a feature by setting them to true/false here
         // for example:
         // this.FEATURE_FLAG_NAME = false;
+        this.NEW_HOTNESS = true;
```

Comment:
remove

Reply:
UGH I thought I didn't have this! Thanks. This is my bad.